src/backend/joanie/debug/views.py

```diff
@@ -79,6 +82,112 @@ class DebugMailSuccessPaymentViewTxt(DebugMailSuccessPayment):
     template_name = "mail/text/order_validated.txt"
 
 
+class DebugMailSuccessInstallmentPaid(TemplateView):
+    """Debug View to check the layout of the success installment payment by email"""
+
+    def get_context_data(self, **kwargs):
+        """
+        Base method to prepare the document context to render in the email for the debug view.
+        Usage reminder :
+            /__debug__/mail/installment_paid_html
+        """
+        product = ProductFactory(price=Decimal("1000.00"))
+        product.set_current_language("en-us")
+        product.title = "Test product"
+        product.set_current_language("fr-fr")
+        product.title = "Test produit"
+        product.save()
+        order = OrderGeneratorFactory(
+            product=product,
+            state=ORDER_STATE_PENDING_PAYMENT,
+            owner=UserFactory(first_name="John", last_name="Doe", language="en-us"),
+        )
+        invoice = Invoice.objects.create(
+            order=order,
+            parent=order.main_invoice,
+            total=0,
+            recipient_address=order.main_invoice.recipient_address,
+        )
+        for payment in order.payment_schedule[:2]:
+            payment["state"] = PAYMENT_STATE_PAID
+            Transaction.objects.create(
+                total=Decimal(payment["amount"].amount),
+                invoice=invoice,
+                reference=payment["id"],
+            )
+        current_language = translation.get_language()
+        with translation.override(current_language):
+            product.set_current_language(current_language)
+            return super().get_context_data(
+                order=order,
+                product_title=product.title,
+                order_payment_schedule=order.payment_schedule,
+                installment_amount=Money(order.payment_schedule[2]["amount"]),
+                product_price=Money(order.product.price),
+                count_installment_paid=order.get_count_installments_paid(),
```

Comment:
```suggestion
                count_installments_paid=order.get_count_installments_paid(),
```

Reply:
I've taken away this variable, the template will handle the installment concerned position from the index with the method that we use for the table of installment rendering.